app/lib/features/settings/pages/settings_page.dart

```diff
@@ -50,35 +51,53 @@ class SettingsPage extends ConsumerWidget {
     final accountInfo = ref.watch(accountAvatarInfoProvider);
     final shouldGoReplacement = context.isLargeScreen;
     final userId = account.userId().toString();
-    return Card(
-      shape: RoundedRectangleBorder(
-        borderRadius: BorderRadius.circular(16),
-      ),
-      child: Column(
-        children: [
-          ListTile(
-            onTap: () => shouldGoReplacement
-                ? context.pushReplacementNamed(Routes.myProfile.name)
-                : context.pushNamed(Routes.myProfile.name),
-            leading: ActerAvatar(
-              options: AvatarOptions.DM(accountInfo),
-            ),
-            title: Text(
-              accountInfo.displayName ?? '',
-              overflow: TextOverflow.ellipsis,
-              style: Theme.of(context).textTheme.titleSmall,
+    return InkWell(
+      onTap: () => shouldGoReplacement
+          ? context.pushReplacementNamed(Routes.myProfile.name)
+          : context.pushNamed(Routes.myProfile.name),
+      child: Card(
+        shape: RoundedRectangleBorder(
+          borderRadius: BorderRadius.circular(16),
+        ),
+        child: Column(
+          children: [
+            ListTile(
+              leading: ActerAvatar(
+                options: AvatarOptions.DM(accountInfo),
+              ),
```

Comment:
Refactorted the UserHeader card a bit for usability:
1. trailing now has the QR-code button
2. the entire area is now clickable to get into the profile edit